waiting-for-start.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "waiting for servers to start..."
+declare -a arr=("npm" "java")
+for pid in ${arr[@]}; do
+  while true; do
+    echo $pid
+    if ! pgrep -n -f $pid > /dev/null; then
+      sleep 1;
+      echo "still waiting..."
+      echo $(pgrep -n -f npm)
+    else
+      echo "All needed background processes start running!"
```

Comment:
Does the appearance of process guarantee the state of the process? A server process might be in the process of starting and not yet ready for connections.